src/assets/JSAsset.js

```diff
@@ -24,6 +28,26 @@ class JSAsset extends Asset {
     this.isAstDirty = false;
     this.isES6Module = false;
     this.outputCode = null;
+
+    // decode .env
+    this.envTable = {};
+    const envRegex = /^(\w+)\s*=\s*([\s\S]+)/;
+    const envFile =
+      process.env.NODE_ENV === 'development' ? ['.dev.env'] : ['.env'];
```

Comment:
This is kind of tricky. `NODE_ENV` is something that was popularized by Express. I know most of us would probably end up using `NODE_ENV`, but it's really not required to do so. If parcel is going to look for that _exact_ prop, then can you also add it in the documentation somewhere?

Reply:
Yes I agree. And I am not sure if it is a good idea seperate `.env` and `.dev.env` this way.

Reply:
@gongpeione as long as it is documented, I think it's okay. 😄

Reply:
How to handle env other than dev and prod? e.g. dev/uat/prod.